Improved readability and detatched some commands from their output (#7324)
Kong/docs.konghq.com: app/_src/gateway-operator/guides/autoscaling-kong.md

            limits:
              memory: "1024Mi"
              cpu: "1000m"

>           # Add here any Konnect related environment variables, volumes etc...

Reviewer: ```suggestion
          # Add any Konnect-related configuration here: environment variables, volumes, and so on.
```

    export PROXY_IP=$(kubectl get dataplanes.gateway-operator.konghq.com -o jsonpath='{.status.addresses[0].value}' horizontal-autoscaling)
    ```

> 1. Install [`k6s`](https://k6.io/) then create a configuration file containing the following code:

Reviewer: ```suggestion
1. Install [`k6s`](https://k6.io/), then create a configuration file containing the following code:
```

1. Start the load test.

   ```
   k6 run k6.js
   ```

1. Observe the scaling events in the cluster while the test is running.

    ```bash
    kubectl get events --field-selector involvedObject.name=horizontal-autoscaling --field-selector involvedObject.kind=HorizontalPodAutoscaler
    ```

>     The output will show the scaling events:

Reviewer: ```suggestion
    The output shows the scaling events:
```
